### DISCUSSION TITLE
Feature request: include "paste into folder" on Deepin File Manager

### DISCUSSION THREAD

**Discussion by JyotiGomes** [linuxdeepin/developer-center#2944] [Features Request & Ideas | 特性请求 & 头脑风暴]
In Deepin File manager, if we cut a file and then select a folder, a directory, and click with the right mouse button to paste into the folder, the option "paste into folder" does not appear. The paste option only appears if we enter the directory. We are unable to paste into a selected directory from the outside. This doesn't happen in any other distro or File Manager. Deepin is wonderful, but why Deepin File Manager doesn't have yet such a basic and useful feature?

**Comment by meiyixiang** [comment]
This is a good suggestion, I have given feedback to the product and hope to support this function as soon as possible
